data/bss-factory-sets.json

```diff
@@ -0,0 +1,5689 @@
+        "nature": "impish",
+        "evs": [252, 0, 252, 0, 0, 4],
+        "teratype": ["water", "dragon"],
+        "ability": "mirrorarmor"
+      }
+    ]
+  },
+  "zapdos": {
+    "weight": 6,
+    "sets": [
+      {
+        "species": "zapdos",
+        "weight": 75,
+        "moves": [["voltswitch"], ["roost"], ["hurricane"], ["discharge"]],
+        "item": ["rockyhelmet", "heavydutyboots", "sitrusberry"],
+        "nature": "bold",
+        "ivs": [31, 0, 31, 31, 31, 31],
+        "evs": [252, 0, 252, 0, 4, 0],
+        "teratype": ["water", "steel"],
+        "ability": "static"
+      },
+      {
+        "species": "zapdos",
+        "weight": 10,
+        "moves": [["voltswitch"], ["roost"], ["hurricane"], ["terablast"]],
+        "item": ["rockyhelmet", "heavydutyboots", "sitrusberry"],
+        "nature": "bold",
+        "ivs": [31, 0, 31, 31, 31, 31],
+        "evs": [252, 0, 252, 0, 4, 0],
+        "teratype": ["ice"],
+        "wantsTera": true,
+        "ability": "static"
+      },
+      {
+        "species": "zapdos",
+        "weight": 10,
+        "moves": [
+          ["thunderbolt"],
+          ["hurricane"],
+          ["voltswitch"],
+          ["terablast"]
+        ],
+        "item": ["choicescarf"],
+        "nature": "timid",
+        "ivs": [31, 0, 31, 31, 31, 31],
+        "evs": [0, 0, 4, 252, 0, 252],
+        "teratype": ["ice", "fairy"],
+        "wantsTera": true,
+        "ability": "static"
+      },
+      {
+        "species": "zapdos",
+        "weight": 5,
+        "moves": [["thunderbolt"], ["hurricane"], ["voltswitch"], ["heatwave"]],
+        "item": ["choicescarf", "choicespecs"],
+        "ivs": [31, 0, 31, 31, 31, 31],
+        "nature": "timid",
+        "evs": [0, 0, 4, 252, 0, 252],
+        "teratype": ["electric"],
+        "ability": "static"
+      }
+    ]
+  },
+  "ursaluna": {
+    "weight": 6,
+    "sets": [
+      {
+        "species": "ursaluna",
+        "weight": 85,
+        "moves": [["facade"], ["earthquake"], ["trailblaze"], ["swordsdance"]],
+        "item": ["flameorb"],
+        "nature": "jolly",
+        "evs": [0, 252, 0, 0, 4, 252],
+        "teratype": ["normal", "water", "grass"],
+        "ability": "guts"
+      },
+      {
+        "species": "ursaluna",
+        "weight": 15,
+        "moves": [["earthquake"], ["avalanche"], ["yawn"], ["protect"]],
+        "item": ["leftovers"],
+        "nature": "impish",
+        "evs": [156, 4, 252, 0, 92, 4],
+        "teratype": ["water", "fairy"],
+        "ability": "bulletproof"
+      }
+    ]
+  },
+  "goodrahisui": {
+    "weight": 6,
+    "sets": [
+      {
+        "species": "goodrahisui",
+        "weight": 60,
+        "moves": [
+          ["flashcannon", "heavyslam"],
+          ["dracometeor", "icebeam"],
+          ["acidspray", "thunderbolt", "flamethrower"],
+          ["earthquake"]
+        ],
+        "item": ["assaultvest"],
+        "nature": "quiet",
+        "evs": [252, 0, 0, 252, 0, 4],
```

Comment:
Please don't put the 4 in Speed on max/max sets in randoms. Move it elsewhere.

Also applies to other Hoodras.